client/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "logux-to-do-example-client",
+  "private": true,
+  "version": "0.0.0",
+  "engines": {
+    "node": ">=16"
+  },
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "lint:css": "stylelint '**/*.css' --fix"
+  },
+  "dependencies": {
+    "@logux/client": "^0.15.1",
+    "@logux/core": "^0.7.3",
+    "@nanostores/react": "^0.1.5",
+    "classnames": "^2.3.1",
+    "nanoid": "^3.3.2",
+    "nanostores": "^0.5.12",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "typescript": "^4.6.3",
+    "wouter": "^2.8.0-alpha.2"
+  },
+  "devDependencies": {
+    "@logux/stylelint-config": "^0.9.0",
+    "@types/react": "^17.0.33",
+    "@types/react-dom": "^17.0.10",
+    "@vitejs/plugin-react": "^1.0.7",
+    "browserslist": "^4.20.2",
+    "postcss": "^8.4.12",
+    "postcss-nested": "^5.0.5",
+    "postcss-normalize": "^10.0.0",
+    "stylelint": "^14.6.1",
+    "vite": "^2.8.0"
```

Comment:
Let’s keep all build dependencies in `dependencies` and put to `devDependenies` only `Stylelint` and types.

There is good advice to install only `--production` dependencies during deploy to avoid security risks from development dependencies. Let’s promote this practices here as well.